docs/gcloud-auth.rst

```diff
@@ -27,6 +27,10 @@ Overview
 
         $ export GOOGLE_APPLICATION_CREDENTIALS="/path/to/keyfile.json"
 
+    The default project ID will be the project that keyfile.json was genereated
+    from. This can be overridded by setting the ``GCLOUD_PROJECT`` environment
+    variable.
```

Comment:
This is the "wrong" order. As @jgeewax mentioned, when they go out of their way to provide an explicit env. var., that is what we should use first.